cypress/e2e/models/migration/archetypes/archetype.ts

```diff
@@ -244,4 +244,36 @@ export class Archetype {
         cy.wait(8 * SEC);
         Assessment.perform_review(risk);
     }
+
+    validateReviewFields(): void {
+        let actionList = [
+            `Archetype - ${this.name}-Rehost`,
+            `Archetype - ${this.name}-Replatform`,
+            `Archetype - ${this.name}-Refactor`,
+            `Archetype - ${this.name}-Retain`,
+            `Archetype - ${this.name}-Repurchase`,
+            `Archetype - ${this.name}-Retire`,
+        ];
+        let effortEstimateList = [
+            `Archetype - ${this.name}-Small`,
+            `Archetype - ${this.name}-Medium`,
+            `Archetype - ${this.name}-Large`,
+            `Archetype -$ {this.name}-Extra Large`,
+        ];
+        let criticalityList = [
```

Comment:
This list seems to be common between archetype and application page . It can be put in assessment.ts and page name(Application/Archetype) can be passed as parameter.

Example :

`${page} -$ {this.name}-Extra Large`